shared/elements/gaia_lowstorage_dialog/examples/locales/example.en-US.properties

```diff
@@ -0,0 +1,7 @@
+lowstorage-dialog-title=Low Device Storage
```

Comment:
__question:__ should not we have ```shared/elements/gaia_lowstorage_dialog/locales/gaia_lowstorage_dialog.en-US.properties``` for the common strings (title, learnmore, ok, generic-test)?